lib/chef/provider/aws_vpc.rb

```diff
@@ -55,6 +55,41 @@ def update_aws_object(vpc)
   end
 
   def destroy_aws_object(vpc)
+    if purging
+      # TODO we need to destroy the associated data bags as well.
```

Comment:
Shouldn't we wait to delete the data bag until after the instance has successfully been deleted (next code block below)?  Also, this needs to be done.